index.js

```diff
@@ -83,8 +83,16 @@ Notifier.prototype.scan = function () {
             bodies: ''
         });
         fetch.on('message', function (msg) {
+            let uid, flags;
```

Comment:
Can you change the `let` to `var` ? Since I'm not using any `let` I don't want to break people's script if they are running it on an old version of node.
I'll be happy to merge it after.
Thanks

Reply:
Ok.Thanks